packages/core/package.json

```diff
@@ -91,8 +92,10 @@
     "@ethersproject/address": "5.6.1",
     "debug": "^4.3.2",
     "ethers": "^5.4.7",
+    "fs-extra": "^10.0.0",
```

Comment:
The file access for the file deployment loader needs this. We could look at simplifying to just `fs` but I am not sure it is worth the trouble.